news/_posts/releases/2020-10-22-prestashop-1-7-7-0-rc-release.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title:  "PrestaShop 1.7.7.0 RC1 Is Ready To Be Tested!"
+subtitle: "A few weeks after the beta, here it is: the first release candidate for PrestaShop 1.7.7.0 is now ready for you to test!"
+date:   2020-10-22 15:00:00
+authors: [ PrestaShop ]
+image: /assets/images/2020/10/build-177RC1.png
+icon: icon-leaf
+tags:
+ - 1.7.7
+ - version
+ - rc
+ - minor
+ - releases
+---
+
+We are really happy to announce that we are close to releasing 1.7.7.0, the release candidate is the last step before releasing the final version. 
+
+![1.7.7.0 RC1 is available!](/assets/images/2020/10/build-177RC1.png)
+
+This release candidate aims at making sure that all major regressions found during the beta have been fixed, and that no new ones have been spotted. If no new critical or major regression is reported by November 6th, we will make the final version of 1.7.7.0 available.
+
+Please note that it is not currently possible to perform an automatic upgrade from the Beta to the Release Candidate. Furthermore, it will not be possible to upgrade from this Release Candidate to the final release. If you perform an update of your test store to the RC version, you will need to rollback to your current version before being able to upgrade to the final 1.7.7.0.
+
+Please download, install and test this new version of PrestaShop and give us your feedback as early as possible. The later a problem is identified, the later it can be fixed and this delays the final output.
+If you find regressions compared to the 1.7.6 version, please open an issue and describe it on [Github](https://github.com/PrestaShop/PrestaShop/issues/new?template=1_bug_report.md) so that it can be reproduced and fixed as soon as possible.
```

Comment:
If you do not insert blank empty lines between statements, it will be rendered as a single block (see https://github.com/PrestaShop/prestashop.github.io/blob/07f6aa7f430024c759427619a312556880d36bad/news/_posts/releases/2020-10-22-prestashop-1-7-7-0-rc-release.md) ; is that intended ?

Reply:
No, thank you I am going to fix that !

Reply:
if you need the equivalent of a `< br/>`, in markdown, add 2 spaces at the end of the line.
https://daringfireball.net/projects/markdown/syntax#p